extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -129,7 +134,7 @@ function BaseCarouselWithRef(
           state management code.
         */}
         {childrenArray.map((child, index) =>
-          Math.abs(index - currentSlide) < 2 ? (
+          Math.abs(index - currentSlide) < visibleCount * 2 ? (
```

Comment:
As mentioned before, let's bump up to `* 3` so we don't forget :)

Reply:
Done